packages/expo-cli/src/commands/build-native/index.ts

```diff
@@ -70,12 +73,136 @@ export async function buildAction(projectDir: string, options: Options): Promise
   }
 }
 
+async function logBuildStatuses(
+  username: string,
+  buildStatus: {
+    jobs: Array<Record<string, any>>;
```

Comment:
1. Rename `jobs` -> `builds`.
2. Add a type definition for builds - `Build` - and list all fields that we want to use in expo-cli.
3. Put the `Build` type definition in a separate file. I'm sure it'll be used for other code in `expo-cli` too.